components/next_steps_view/steps.ts

```diff
@@ -140,6 +144,16 @@ export const Steps: StepType[] = [
             defaultMessage: 'Next step',
         },
     },
+    {
+        id: RecommendedNextSteps.CREATE_FIRST_CHANNEL,
+        title: {
+            titleId: t('next_steps_view.titles.createChannel'),
+            titleMessage: 'Create your first channel',
+        },
+        roles: ['system_admin'],
```

Comment:
> Just some questions on top of this one; It looks like in the description of the ticket, after this step we need to start the tutorial tips. That means we need to ensure this new step is always the last to be completed. Are we doing that?
> 
> I also think we need to updated the E2E tests to include this step

* Still waiting for the definition if this actually needs to be only shown to the first admin, but it would be just matter of changing the role.
* Good point, so, yes, by adding the new step as the last step in the array it should make it to be the last one. However, once this A/B testing end and we conclude which one will be the permanent option, we can apply a more safest way of making sure this step is the last one.
* Same thing about the A/B this still being an A/B test. This PR fixes the tests that broke/modified because of this changes and even add unit testing for steps, tips and for the create new channel component. However, not sure if it is worth the effort to add new e2e for this. Maybe once it has decided which will be the final option it will make more sense.

Reply:
@annelieseklein can you confirm this is should be available for all admins?

Reply:
@marianunez yes available for all admins